drafting/conda-template.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# This script reads in a yaml file (qiime2 recipe)
+# Pulls the needed data from that file
+# And uses that data to write to a secondary yaml file
+# Which will be the end result: a conda recipe
+
+import yaml
+import os
+import sys
+
+if __name__ == '__main__':
+    # Reading the qiime2 recipe file
+    qiime2_recipe = sys.argv[1]
+    if sys.argv[1] == None:
+        raise ValueError('Missing required parameter. QIIME 2 recipe filepath must be provided.')
+
+    with open(qiime2_recipe) as qiime2_recipe:
+        parsed_recipe = yaml.load(qiime2_recipe, Loader=yaml.FullLoader)
+
+    filepath = sys.argv[2]
+    if sys.argv[2] == None:
+        raise ValueError('Missing required parameter. conda recipe filename must be provided.')
```

Comment:
The same concern as above applies here, this will generate an index error if the argument isn't provided, so that means your nice error message here will _never_ be shown to the caller! 😢